linera-chain/src/chain.rs

```diff
@@ -235,6 +235,13 @@ impl BundleInInbox {
     }
 }
 
+// The number of timestamp in a bucket
+// The `TimestampedBundleInInbox` type contains 4 cryptohashes, 1 blockheight
+// an index, two enums and the ChannelName. Only the ChannelName has an unbounded
+// size but we can expect the size to be reasonably small, so a total size of 100
+// seems reasonable for the storing of the data.
+const TIMESTAMPBUNDLE_BUCKET_SIZE: usize = 100;
```

Comment:
`UNSKIPPABLE_BUNDLES_BUCKET_SIZE`

Reply:
We could also decide not create a constant altogether.

@afck What do you think?

Reply:
You mean just inline the `100`? Or make it configurable? I think this is an example where I wouldn't make it configurable, since this is very hard to explain to the user and there is probably simply a range of reasonable values one of which we can fix.

Reply:
Yes, I meant if inlining the value wouldn't be just fine. Changing the value breaks the storage format -- which is internal to a validator but still...